node/lib/util/config_util.js

```diff
@@ -59,16 +59,31 @@ exports.getConfigString = co.wrap(function *(config, key) {
     return null;
 });
 
+
+/**
+ * Attempt to set the key, value pair in the specified git configuration.  Returns 0 or an error code
+ * @param {NodeGit.Config} config
+ * @param {String}         key
+ * @param {Integer}        value
+ *
+ */
+exports.setConfigInt = co.wrap(function *(config, key, val) {
```

Comment:
I do not see what this adds over callers just  calling config.setInt64?

Reply:
Fixed